recipes/libcurl/all/conanfile.py

```diff
@@ -363,21 +363,25 @@ def package(self):
             self.copy(pattern="*.def", dst="lib", keep_path=False)
             self.copy(pattern="*.lib", dst="lib", keep_path=False)
 
+        self.copy("cacert.pem", dst="res")
+
         # no need to distribute share folder (docs/man pages)
         shutil.rmtree(os.path.join(self.package_folder, 'share'), ignore_errors=True)
-        # no need for pc files 
+        # no need for pc files
         shutil.rmtree(os.path.join(self.package_folder, 'lib', 'pkgconfig'), ignore_errors=True)
-        # no need for cmake files 
+        # no need for cmake files
         shutil.rmtree(os.path.join(self.package_folder, 'lib', 'cmake'), ignore_errors=True)
         # Remove libtool files (*.la)
         if os.path.isfile(os.path.join(self.package_folder, 'lib', 'libcurl.la')):
             os.remove(os.path.join(self.package_folder, 'lib', 'libcurl.la'))
+        # library package should not contain executable
+        shutil.rmtree(os.path.join(self.package_folder, 'bin'), ignore_errors=True)
```

Comment:
why this? I think that this recipe generates `curl` executable too. Where should it go then?